### PR TITLE
refactor: styling and cleanup

### PR DESCRIPTION
**Pull request by favna**

### CODE REVIEW

File: .yarn/patches/shiki-es-npm-0.2.0-a9934bd6d5.patch
Comment (favna) [inline .yarn/patches/shiki-es-npm-0.2.0-a9934bd6d5.patch:1]: ref: https://github.com/unjs/shiki-es/pull/7

File: nuxt.config.ts [hunk 1]
```diff
@@ -59,9 +59,27 @@ const manifestIcons = [
...
+		preset: 'cloudflare-pages',
+		prerender: {
+			routes: ['/', '/about', '/sitemap.xml']
+		}
+	},
+	vite: {
+		define: {
+			'process.env.VSCODE_TEXTMATE_DEBUG': 'false'
+		}
+	},
```
Comment (kyranet) [inline nuxt.config.ts:74]: 🧹
```suggestion
```

File: nuxt.config.ts [hunk 2]
```diff
@@ -70,7 +88,7 @@ export default defineNuxtConfig({
 		},
 		workbox: {
 			navigateFallback: '/',
-			globPatterns: ['**/*.{js,css,html,png,svg,ico,xml}']
+			globPatterns: ['**/*.{js,css,html,png,svg,ico,xml,json,wasm}']
```
Comment (kyranet) [inline nuxt.config.ts:91]:
```suggestion
			globPatterns: ['**/*.{js,css,html,png,svg,ico,xml}']
```

File: package.json [hunk 3]
```diff
@@ -21,6 +18,7 @@
 		"@vite-pwa/nuxt": "^0.0.8",
 		"@vueuse/core": "^10.1.2",
 		"@vueuse/nuxt": "^10.1.2",
+		"canvas-constructor": "^7.0.1",
```
Comment (kyranet) [inline package.json:21]: This doesn't trigger a change in yarn.lock? I thought it did.

**Review by kyranet — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (favna) [reply package.json:21]: Nope. Unlock `package-lock.json`, yarn doesnt track whether something is a dev or prod dependency. It references the `package.json` for that.. as it should, why keep 2 sources of truth.

**Review by kyranet — APPROVED** [review]

**Pull request merged**